plasma_framework/test/src/vaults/Erc20Vault.test.js

```diff
@@ -0,0 +1,138 @@
+const BlockController = artifacts.require('BlockController');
+const Erc20Vault = artifacts.require('Erc20Vault');
+const ERC20 = artifacts.require('ERC20Mintable');
+const BadERC20 = artifacts.require('BadERC20');
+
+const { BN, expectRevert } = require('openzeppelin-test-helpers');
+const { expect } = require('chai');
+
+const Testlang = require('../../helpers/testlang.js');
+const { PaymentTransaction, PaymentTransactionOutput } = require('../../helpers/transaction.js');
+
+contract('Erc20Vault', (accounts) => {
```

Comment:
I do like this, but then the linter complains about `_` being an unused variable 😞 
It would be nice if it followed Elixir's standard of ignoring vars that start with `_`

Reply:
ah...okay : ( didn't aware about that.